gradle/gradle-publish.gradle

```diff
@@ -0,0 +1,81 @@
+apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
+
+ext["signing.keyId"] = System.getenv('SIGNING_KEY_ID') ?: ''
+ext["signing.password"] = System.getenv('SIGNING_KEY_PASSWORD') ?: ''
+ext["signing.secretKeyRingFile"] = System.getenv('SIGNING_KEY_FILE') ?: ''
+ext["SONATYPE_NEXUS_USERNAME"] = System.getenv('SONATYPE_NEXUS_USERNAME') ?: ''
+ext["SONATYPE_NEXUS_PASSWORD"] = System.getenv('SONATYPE_NEXUS_PASSWORD') ?: ''
+
+
+afterEvaluate {
+    android.libraryVariants.each { variant ->
+        def variantName = variant.name
+        def resolvedVersion = project.version
+        publishing.publications.create variantName, MavenPublication, {
+            println "$variantName resolvedVersion: ${resolvedVersion}  ${group} ${project.name}"
```

Comment:
Do we want this `println`? Is it printing to the GH actions console?

Reply:
I added this to debug, i'll remove it